marklogic-data-hub-central/ui/src/components/explore/graph-vis-explore/graph-vis-explore.tsx

```diff
@@ -301,16 +300,16 @@ const GraphVisExplore: React.FC<Props> = (props) => {
     },
   };
 
-  const menuClick = async (event) => {
+  /*   const menuClick = async (event) => {
```

Comment:
@lucianaabc-ml Let's add a comment here mentioning the ticket number DHFPROD-8284